arm/authorization/classicadministrators.go

```diff
@@ -75,7 +75,8 @@ func (client ClassicAdministratorsClient) ListPreparer(apiVersion string) (*http
 		"api-version": APIVersion,
 	}
 
-	return autorest.Prepare(&http.Request{},
+	req := http.Request{}
+	return autorest.Prepare(&req,
```

Comment:
http.Request Cancel channel can be used now to ignore polling for long-running.  So, request will be like...

 ```req := http.request{} ``` for non-lro 

```req := http.request{ Cancel:cancel } ``` for lro